README.md

```diff
@@ -0,0 +1,31 @@
+# UT ECLAIR Smart Mirror Project
+This is the GitHub repository for ECLAIR's smart mirror project. Ultimately, this software will run on a Raspberry Pi and provide a useful interface so users can see the time, any todo items, the weather, etc. through a mirror.
+
+These instructions are not very thorough, so if you have any trouble and the linked details aren't sufficient, ask for clarification on the Discord server.
+
+## Getting Started
+1. Make sure you're on the ClickUp to look at what tasks are being worked on.
+2. Use GitHub to add your features/widgets and start a pull request so Sahil can review it. More details in the following section.
+3. Test your code using SSH. Credentials are on the Discord server. Detailed instructions are in the **Setting up Remote Development** section below.
+
+## Contributing to the Repository
+We're using GitHub, so we'll follow its [collaborative development model](https://docs.github.com/en/github/collaborating-with-pull-requests/getting-started/about-collaborative-development-models). Here are a brief list of steps you'll take to create your first pull request. Each step has a link to more detailed instructions on GitHub's docs.
+1. First, you need to [fork this repository](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/about-forks) so you can make your own changes. At the top right of this page, there's a fork button. Press that, and you'll end up with a copy of this repo.
+2. Now that you have a fork, you're almost ready to start making changes. [Create a new branch](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) in your repository, and name it whatever you'd like, such as the name of the feature you're about to start work on.
+3. Now you need to [clone your repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your computer and actually begin making changes. 
+4. Switch to the branch you just created using `git checkout <branch-name>`. You can use VSCode or some other IDE to make changes. You might not have all the dependencies installed so there could be a lot of errors. You will be able to test your code on the Raspberry Pi using the sections below.
+5. After you're done making changes or at any frequency you'd like to "checkpoint" your code, run `git commit -am "<describe your changes>"` and `git push origin <branch-name>` to send the code to your forked repository.
+6. Once you're done with a feature or task and you'd like it to be reviewed, head back to GitHub and start a [Pull Request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). You can see exactly what changes you made and describe them. Once you're done, you can finish by pressing "Create Pull Request".
```

Comment:
After this step, add a section describing the requirements for a pull request. This includes adding me for review, a image of things working, a proper title, bugs/problems you faced, what to watch out for, limitations on the module, etc.